spec/commands/viz_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe "bundle viz" do
+  before :each do
+    `gem install ruby-graphviz`
+  end
+
+  context "with a standard Gemfile" do
+    before :each do
+      `gem install httpi -v "= 2.2.7"` # requires any rack
+      `gem install rack -v "= 1.1.1"`
+      # TODO: this whole approach is bad
```

Comment:
I struggled with getting this test set up correctly.

There are a bunch of problems here, beyond what this comment is about:

1. The test output can contain a _ton_ of warnings from `ruby-graphviz`'s output, like: "warning: duplicated key at line 986 ignored: "red".
2. I'm using backticks to install gems into the `system_gem_path`, so that they're available to `install_gemfile`. This is because I don't understand the `spec/` structure enough to get the necessary test conditions in place. That means there's an actual gem install happening, I believe. I spent a ton of time trying to find this pair of `httpi` and `rack` — they install quickly enough, with no native extensions, etc., but this can probably be improved.
3. I needed to `gem install ruby-graphviz` in the outer `before`, so that I could get `bundle viz` working. Again, same problem as (2), really.